Gem/Code/Include/MatchPlayerCoinsBus.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+
+namespace MultiplayerSample
+{
+    class MatchPlayerCoinsRequests : public AZ::EBusTraits
```

Comment:
just curious: why use `AZ::EBusTraits` instead of `AZ::ComponentBus` ?

Reply:
AZ::ComponentBus is for case when you will have multiple handlers on different entities. In this case, there is going to be only a single handler, so basically a singleton. So for this one, you can do ::BusConnect() without any specifying any ids. With AZ::ComponentBus you have to specify an entity id when calling ::BusConnect(id);